deploy/openshift/deploy-kiali-to-openshift.sh

```diff
@@ -146,6 +146,8 @@ export GRAFANA_URL="${GRAFANA_URL:-http://grafana-istio-system.127.0.0.1.nip.io}
 export VERBOSE_MODE="${VERBOSE_MODE:-3}"
 export KIALI_USERNAME_BASE64
 export KIALI_PASSPHRASE_BASE64
+export AUTH_STRATEGY="${AUTH_STRATEGY:-login}"
+export OAUTH_SECRET="${OAUTH_SECRET:-kiali-oauth}"
```

Comment:
Is this OAUTH_SECRET an actual credential that is confidential? If not, put it in the SETTINGS output down below in that group of lines starting at 190. Also echo the AUTH_STRATEGY too, so we can see what the value is for debugging customer install issues.

Reply:
We don't actually use or need the oauth secret. I will remove it.